app/routes/_gcn.circulars.$circularId.($version)[.]json.ts

```diff
@@ -10,12 +10,18 @@ import invariant from 'tiny-invariant'
 
 import { formatCircularJson } from './_gcn.circulars/circulars.lib'
 import { get } from './_gcn.circulars/circulars.server'
-import { origin } from '~/lib/env.server'
+import { feature, origin } from '~/lib/env.server'
 import { getCanonicalUrlHeaders } from '~/lib/headers.server'
 
-export async function loader({ params: { circularId } }: DataFunctionArgs) {
+export async function loader({
+  params: { circularId, version },
+}: DataFunctionArgs) {
   invariant(circularId)
-  const result = await get(parseFloat(circularId))
+
+  const result = await get(
+    parseFloat(circularId),
+    version && feature('CIRCULAR_VERSIONS') ? parseFloat(version) : undefined
+  )
```

Comment:
If the `CIRCULAR_VERSIONS` feature is disabled and the `version` route param is defined, then we should throw a 404 error.

Reply:
Resolved in latest